app/assets/stylesheets/application.scss

```diff
@@ -34,3 +65,77 @@ border-radius: 15px;
 color: $black;
 width: 200px;
 }
+
+h4 {
+  background: rgba(65, 92, 108, 0.5);
+  color: $white;
+  font: bold 24px/45px Helvetica, Sans-Serif;
+  font-variant: small-caps;
+  letter-spacing: 6px;
+  padding: 10px;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: 4px;
+}
+
+.container {
+  border: 30px;
+  margin-top: 20px;
+
+  h1 {
+    background: rgba(0, 0, 0, 0.6);
+    border-radius: 15px;
```

Comment:
Properties should be ordered background, -moz-border-radius, -webkit-border-radius, border-radius, color, font, font-variant, letter-spacing, padding, text-align, width